packages/components/src/components/Button/Button.tsx

```diff
@@ -5,16 +5,16 @@ import { Box } from "../../primitives/Box";
 import { Icon } from "../Icon";
 
 type ButtonSizeOptions = "large" | "small";
-type ButtonVariantOptions = "primary" | "secondary" | "text";
+type ButtonVariantOptions = "ghost" | "primary" | "secondary";
 
 type IconNames = keyof typeof HeroOutlineIcons;
 
 export interface ButtonProps
   extends Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, "color"> {
   /** Sets the render element of the component. Either 'a' or 'button'.*/
-  as?: "a" | "button";
-  /** Add a description comment for each prop. */
-  children: NonNullable<React.ReactNode>;
+  as?: "a" | "button" | "label";
```

Comment:
Is this semantically correct? Having a button as a label?

Reply:
I don't think a button should ever render as a label.

Reply:
The only reason to add it is to use it on the upload component. To be accessible, the input should have a label, but in our case, the label looks like a button. I don't think it will be used this way in another place. Do we have another way to style a label likes a button?

Reply:
I never saw it, but I just did a google and found that MUI is using buttons as labels as well https://mui.com/material-ui/react-button/#upload-button. So, I think it's a usual thing to do.